internal/conf/jwk.go

```diff
@@ -0,0 +1,135 @@
+package conf
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/golang-jwt/jwt/v5"
+	"github.com/lestrrat-go/jwx/v2/jwk"
+)
+
+type JwtKeysDecoder map[string]JwkInfo
+
+type JwkInfo struct {
+	PublicKey  jwk.Key `json:"public_key"`
+	PrivateKey jwk.Key `json:"private_key"`
+}
+
+// Decode implements the Decoder interface
+// which transforms the keys stored as der binary strings into jwks
```

Comment:
check if comment is still relevant 
```suggestion
```

Reply:
Are they still stored as DER? Maybe a stray comment?